src/program.rs

```diff
@@ -108,6 +119,22 @@ impl Program {
         self.render()
     }
 
+    /// Returns new index to advance to
+    pub fn skip_forward(&mut self) -> Result<(), String> {
+        let skip_size = compute_skip_size(&self.images);
+        self.increment(skip_size)?;
+        self.render()?;
```

Comment:
Increment already includes a call to render. So we can just go with `self.increment(skip_size)` without the `?;` and also get rid of the `Ok(())`

Reply:
Yeah... I keep forgetting that increment and decrement also render the new ui state.